google_cloud_speech.py

```diff
@@ -0,0 +1,20 @@
+# Import the speech recognition library
+import speech_recognition as sr
+
+# Create a recognizer
+r = sr.Recognizer()
+
+# Create a microphone objec that gets system default
+mic = sr.Microphone()
+
+# Listen for audio and save it
+with mic as source:
+    audio = r.listen(source)
+
+GOOGLE_CLOUD_SPEECH_CREDENTIALS = /Users/grahamderry/praxis-road-272622-d0ad9ed1867c.json
```

Comment:
The path to the JSON file probably needs to be in quotes.

Reply:
Actually it says the path to the JSON file needs to be in an environment variable. An environment variable is something you usually set in the terminal (which has to be set every time you want to run the script and open a new terminal. In the terminal you type `export GOOGLE_CLOUD_SPEECH_CREDENTIALS='<MY_PATH>'`

Reply:
Another could thing to do is create a file called app-env (or whatever) in which you have a bunch of one liners like the one above (for multiple things in environment variables). Then you simply `source app-env` every time before running your script.